src/pages/home/report/ReportActionCompose.js

```diff
@@ -566,12 +567,14 @@ class ReportActionCompose extends React.Component {
                             </>
                         )}
                     </AttachmentModal>
-                    <EmojiPicker
-                        isDisabled={isBlockedFromConcierge || isArchivedChatRoom}
-                        onModalHide={() => this.focus(true)}
-                        onEmojiSelected={this.addEmojiToTextBox}
-                        onBeforeShowEmojiPicker={() => this.textInput.blur()}
-                    />
+                    {canUseTouchScreen() && !this.props.isSmallScreenWidth ? null : (
```

Comment:
didn't use isMediumScreenWidth as it's false for bigger tablets

Reply:
@thesahindia So I guess we could use `isMediumScreenWidth` here and accept that on bigger tablets the emoji will be visible. The core problem in this issue is that on **_small_** tablets, the UI looks bad.

Reply:
Sorry for the delay @thesahindia, I got a bit confused now, we are using the `isSmallScreenWidth` already which would be what is natural to use. 

Just to confirm, if we use this, then in portrait mode, tablets both still have the emoji picker shown ie the width is not the longer side always but it changes based on orientation?